### ISSUE TITLE
Problem: MCPclient ext. package deps fails on new ubuntu14 deploys

### ISSUE THREAD

**Issue by mamedin** [artefactual-labs/ansible-role-archivematica-src#176]
On new AM 1.7 deployments with ubuntu 14 the "Install MCPClient ext. package deps." task fails (p.e OVH Cloud VMs):

```
    "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"     install 'ffmpeg'' failed: E: Unable to correct problems, you have held broken packages.\n",
    "stderr": "E: Unable to correct problems, you have held broken packages.\n",
    "stderr_lines": [
        "E: Unable to correct problems, you have held broken packages."
    ],
    "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nSome packages could not be installed. This may mean that you have\nrequested an impossible situation or if you are using the unstable\ndistribution that some required packages have not yet been created\nor been moved out of Incoming.\nThe following information may help to resolve the situation:\n\nThe following packages have unmet dependencies:\n ffmpeg : Depends: libavdevice57 (>= 7:3.3.2) but it is not going to be installed\n          Depends: libavfilter6 (>= 7:3.3.2) but it is not going to be installed or\n                   libavfilter-extra6 (>= 7:3.3.2) but it is not going to be installed\n",
    "stdout_lines": [
        "Reading package lists...",
        "Building dependency tree...",
        "Reading state information...",
        "Some packages could not be installed. This may mean that you have",
        "requested an impossible situation or if you are using the unstable",
        "distribution that some required packages have not yet been created",
        "or been moved out of Incoming.",
        "The following information may help to resolve the situation:",
        "",
        "The following packages have unmet dependencies:",
        " ffmpeg : Depends: libavdevice57 (>= 7:3.3.2) but it is not going to be installed",
        "          Depends: libavfilter6 (>= 7:3.3.2) but it is not going to be installed or",
        "                   libavfilter-extra6 (>= 7:3.3.2) but it is not going to be installed"
```

**Comment by mamedin** [comment]
An apt cache refresh is needed after adding 2 new apt repositories in pipeline-osdeps-MCPClient.yml.

**Comment by mamedin** [comment]
The problem is not the cache refresh. The ffmpeg dependency issue is fixed when all these packages are installed:

https://github.com/artefactual/archivematica/blob/stable/1.7.x/src/MCPClient/osdeps/Ubuntu-14.json

When running the playbook again it is fixed.

**Comment by mamedin** [comment]
Fixed on #190
